test/parallel/test-tls-peer-certificate.js

```diff
@@ -49,6 +54,28 @@ connect({
     peerCert.fingerprint,
     '8D:06:3A:B3:E5:8B:85:29:72:4F:7D:1B:54:CD:95:19:3C:EF:6F:AA'
   );
+  assert.strictEqual(
+    peerCert.fingerprint256,
+    'A1:DC:01:1A:EC:A3:7B:86:A8:C2:3E:26:9F:EB:EE:5C:A9:3B:BE:06' +
+    ':4C:A4:00:53:93:A9:66:07:A7:BC:13:32'
+  );
+
+  /* SHA256 fingerprint of the public key */
```

Comment:
`//` style comments, please.  I'd wrap the checks in `{ ... }` blocks so you can say `const sha256 = ...`.